src/walletSdk/Customer/index.ts

```diff
@@ -26,14 +31,36 @@ export class Sep12 {
     };
   }
 
-  async getByIdAndType(id: string, type: string): Promise<GetCustomerResponse> {
+  async getCustomer({
+    id,
+    type,
+    memo,
+    lang,
+  }: GetCustomerParams): Promise<GetCustomerResponse> {
+    if (!id && !type && !memo) {
+      throw new NoGetCustomerParamError();
+    }
+    const queryParams: string[] = [];
+    if (type) {
+      queryParams.push(`type=${type}`);
+    }
+    if (id) {
+      queryParams.push(`id=${id}`);
+    }
+    if (memo) {
+      queryParams.push(`memo=${memo}`);
+    }
+    if (lang) {
+      queryParams.push(`lang=${lang}`);
+    }
+    const queryString = queryParams.join("&");
```

Comment:
@acharb hey, so the `queryString` lib has this very cool method which we could use to replace this chunk of code in case you'd like to take a look.

You could simply do something like this:

```
import queryString from "query-string";

async getCustomer(params: GetCustomerParams): Promise<GetCustomerResponse> {

   const qs = queryString.stringify(params);
   ...
   `${this.baseUrl}/customer?${qs}`
   ...
}
```

Reply:
ahh nice I didn't know we had query-string installed already much cleaner
https://github.com/stellar/typescript-wallet-sdk/pull/71/commits/f58a1895a258b6c115e1beec6601f30e80f284e4